static/bidder-params/facebook.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This schema doesn't have fields for required params. Is this realistic? If so, is it safe to assume in this case that there aren't any required params?

Reply:
oops... no, this is a bug. Good catch!

It should mark `placementId` as required.